test/controllers/ModulesSpec.scala

```diff
@@ -52,11 +63,10 @@ object ModulesSpec extends PlaySpecification with Mockito {
     }
   }
 
-  def createModules: (Modules, ModulesLookup, ModuleDao) = {
-    val lookup = mock[ModulesLookup]
-    val dao = mock[ModuleDao]
-    val modules = new Modules(lookup, dao)
+  def createModules(app: play.api.Application): (Modules, ModulesLookup, ModuleDao) = {
+    val lookup = app.injector.instanceOf[ModulesLookup]
```

Comment:
It is not more difficult since we have `Helpers.stubControllerComponents` method. But in this case, since we already have an `Application`,  is more practical to use it. Also, mixing application objects with mocks makes no sense here IMO.